compiler/rustc_transmute/src/lib.rs

```diff
@@ -54,6 +51,10 @@ pub enum Reason {
     DstIsPrivate,
     /// `Dst` is larger than `Src`, and the excess bytes were not exclusively uninitialized.
     DstIsTooBig,
+    /// Src has to have a stricter algignment than Dst, but it does not.
+    InvalidAlignment,
```

Comment:
Keeping with the pattern of variant names that are a little more explanatory:
```suggestion
    DstHasStricterAlignment,
```